library/core/src/main/java/com/google/android/exoplayer2/Format.java

```diff
@@ -572,22 +599,23 @@ public static Format createSampleFormat(
     }
     drmInitData = in.readParcelable(DrmInitData.class.getClassLoader());
     metadata = in.readParcelable(Metadata.class.getClassLoader());
+    isWideAspectRatio = Util.readBoolean(in);
   }
 
   public Format copyWithMaxInputSize(int maxInputSize) {
     return new Format(id, containerMimeType, sampleMimeType, codecs, bitrate, maxInputSize, width,
         height, frameRate, rotationDegrees, pixelWidthHeightRatio, projectionData, stereoMode,
         colorInfo, channelCount, sampleRate, pcmEncoding, encoderDelay, encoderPadding,
         selectionFlags, language, accessibilityChannel, subsampleOffsetUs, initializationData,
-        drmInitData, metadata);
+        drmInitData, metadata, true);
```

Comment:
All of these copy methods need to copy whatever the value of the field is in this instance, rather than hardcoding it to true as part of the copy step.